assets/stylesheets/modules/ai-helper/common/ai-helper.scss

```diff
@@ -314,7 +314,7 @@
   }
 
   .showing-ai-suggestions {
-    #reply-title {
+    #reply-title:not([disabled]) {
```

Comment:
Do we need the CSS changes now that the elements aren't rendered when they are marked as disabled? I may be misunderstanding...